src/assets/css/_settings.scss

```diff
@@ -16,12 +16,8 @@ $c-red-light: #FFBBAC;
 $c-yellow-light: #ffe6be;
 $c-blue-light: #A3E6FD;
 
-// fonts & text
-
-@import url('https://fonts.googleapis.com/css?family=Eczar|Work+Sans:300');
-
 // font families
-$brand-font: 'Eczar', serif;
+$brand-font: 'Eczar', Georgia, serif;
```

Comment:
@rudigier, does this make sense? Looks nicer than Times New Roman if Google Fonts is slow IMHO